### DISCUSSION TITLE
Cannot get user SSO link

### DISCUSSION THREAD

**Discussion by atnartur** [chatwoot/chatwoot#4499] [Q&A] [answered]
I created a platform app and copied access token from super admin interface. After I fix the bug with `underscores_in_headers` in nginx I sent [get sso URL request](https://www.chatwoot.com/developers/api/#operation/get-sso-url-of-a-user) and got `{"error":"Non permissible resource"}` response. What should I do to fix the problem? Chatwoot docker image is already up to date.

**Comment by atnartur** [comment]
With v2.4.0 it does not work, too

**Comment by sojan-official** [comment, marked as answer]
@atnartur Platform apps are only able to access the accounts that they created. So if you are trying to access an account / user which wasn't created using this specific platform app then you will see the error {"error":"Non permissible resource"}

**Reply by atnartur** [reply]
Users from my project will login to Chatwoot for working with clients. When I read API documentation I planned to create users via platform API and get an SSO link for authorization later. But instead I had to add some rows directly to chatwoot's database which is not a good practice, I think.

I suggest to add a retrieving SSO login link permission for platform app tokens. It would be a good way out of the described situation.

Also I created a PR #4543 with addition in API docs to describe current behaviour.
